src/osmf/js/osmf_flash.js

```diff
@@ -90,7 +90,7 @@
     }
     this.encodings = testForFlash();
     this.technology = "flash";
-    this.features = []; //when CC need to be enabled, it needs to be set to OO.VIDEO.FEATURE.CLOSED_CAPTIONS
+    this.features = ["closedCaptions"];
```

Comment:
We have hard coded this value as html5-commom is not loaded in the file. This will be resolved when PBF-471 is fixed.